lms/djangoapps/course_api/blocks/transformers/__init__.py

```diff
@@ -44,6 +45,8 @@ def __init__(
     # 'student_view_multi_device'
     SupportedFieldType(StudentViewTransformer.STUDENT_VIEW_MULTI_DEVICE, StudentViewTransformer),
 
+    SupportedFieldType('special_exam', MilestonesTransformer),
```

Comment:
Can we just call this ``exam_info`` and have all exam information in one place? The distinction between regular exams, special exams, timed exams, and proctored exams is not obvious to me (and I imagine to others). It would be good to keep this API as simple as possible so that clients don't have to understand these distinctions either. I understand if this isn't possible due to the complicated way special exams have been implemented.

If we do want to keep this only to special exams, I'd still rename it to ``special_exam_info``.